src/Tooltip.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+class Tooltip extends Component {
+  componentDidMount () {
+    $('.tooltipped').tooltip();
```

Comment:
You mean using 'ref': (obj) => { this._tooltipped = obj; } to specify the element?
It will be buggy when the child is created by other component.
The data-tooltip-id will be placed outside of element props.